backend/api/consumers.py

```diff
@@ -201,6 +196,7 @@ def get_printer(self):
 
     @newrelic.agent.background_task()
     @close_on_error
+    @close_on_error(exc_class=Printer.DoesNotExist, sentry=False)
```

Comment:
Do you think `OctoprintTunnelWebConsumer` needs similar change? If I remember correctly, it also relies on the db query in "connect" to authenticate.

Reply:
Yes. Adding.